lib/pure/strutils.nim

```diff
@@ -265,8 +352,10 @@ func capitalizeAscii*(s: string): string {.rtl, extern: "nsuCapitalizeAscii".} =
   runnableExamples:
     doAssert capitalizeAscii("foo") == "Foo"
     doAssert capitalizeAscii("-bar") == "-bar"
-  if s.len == 0: result = ""
-  else: result = toUpperAscii(s[0]) & substr(s, 1)
+  if s.len == 0:
```

Comment:
avoid style changes in unrelated code

Reply:
Ah, sorry, it must have been a reflex I changed that... It was next to some code I changed...
Will make sure not to do that! Thanks